lib/components/SideBlock/SideBlock.tsx

```diff
@@ -61,9 +63,12 @@ function SideBlock(props: SideBlockProps) {
         {info && <Info data={info} />}
       </div>
       {children}
-      <div className='dropdown-lines-2 side-block-description'>
+      <SpanTooltip
+        extraClasses='side-block-description dropdown-lines-2'
+        additionalData={description}
```

Comment:
why do we need the additionalData here? the tooltip with description should appear itself once the long text is cut out

Reply:
Actually it is not working for me without it, I guess that something with the calculation of the `scrollWidth` is not perfect?

Reply:
I've added support for `multiLine`, as it was planned to support only single line, but it can be useful